indico/web/client/js/react/components/notes/ConflictModal.jsx

```diff
@@ -0,0 +1,188 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import moment from 'moment/moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  Header,
+  Modal,
+  Button,
+  Message,
+  Icon,
+  Grid,
+  GridRow,
+  GridColumn,
+  List,
+  Segment,
+  ButtonGroup,
+} from 'semantic-ui-react';
+
+import {Param, Translate} from 'indico/react/i18n';
+import {camelizeKeys} from 'indico/utils/case';
+
+import './ConflictModal.module.scss';
+
+export function ConflictModal({noteData, onClose}) {
+  // camelize the noteData keys
+  const note = camelizeKeys(noteData);
+
+  const renderDeletedRevision = () => {
+    return (
+      <Segment placeholder styleName="revision-deleted">
+        <Header icon>
+          <Icon name="trash outline alternate" />
+          <Translate>This content has been deleted.</Translate>
+        </Header>
+      </Segment>
+    );
+  };
+
+  const renderLastModifiedDt = dt => {
+    return (
+      <span title={moment(dt).format('L LTS')} styleName="last-modified-text">
+        {moment(dt).fromNow()}
+      </span>
+    );
+  };
+
+  return (
+    <Modal
+      open
+      onClose={() => onClose(null)}
+      style={{minWidth: '65vw'}}
+      closeOnEscape={false}
+      closeOnDimmerClick={false}
+      closeIcon
+    >
+      <Modal.Header>
+        <Translate>Conflict Detected</Translate>
+      </Modal.Header>
+      <Modal.Content>
+        <Message warning icon>
+          <Icon name="random" />
+          <Message.Content>
+            <Message.Header>
+              <Translate>There were changes made by someone else</Translate>
+            </Message.Header>
+            <p>
+              <Translate>
+                These minutes were last modified{' '}
```

Comment:
```suggestion
                Last modified:{' '}
```

(more neutral language for future adaptability)

Reply:
I see what you did there :sweat_smile: